draft-ietf-lamps-dilithium-certificates.md

```diff
@@ -739,6 +739,13 @@ we show the seed-only format (using a context-specific `[0]` primitive
 tag with an implicit encoding of OCTET STRING), the expanded format,
 and both formats together.
 
+NOTE: All examples use the same seed value, showing how the same seed
+produces different expanded keys for each security level. The seed-only
+format is the most compact representation and is RECOMMENDED for storage.
+The expanded format might be used in constrained environments where key
+expansion is not desired, and the both format allows for key validation
+during import.
```

Comment:
```suggestion
produces different expanded keys for each security level.
```

I think that we should trim this note down. The requirements are found elsewhere in the draft.

Reply:
Also, the note in the same in the ML-KEM draft [PR](https://github.com/lamps-wg/kyber-certificates/pull/102); if we make this change here I will make the same change over there.

Reply:
However a problem remains, in that the initial description of the three formats still emphasises the secondary aspect of the `both` format:
https://github.com/lamps-wg/dilithium-certificates/blob/67b52e5c7cfbc7369c7ef27ddb4f83d5fd18489b/draft-ietf-lamps-dilithium-certificates.md?plain=1#L510-L511
It should instead note that `both` is for interoperability with a broad range of consumers, some that may want to use and retain the seed, others that may only support expanded keys.